discord/enums.py

```diff
@@ -1088,6 +1090,15 @@ class SubscriptionStatus(Enum):
     inactive = 2
 
 
+class ThreadArchiveDuration(IntEnum):
```

Comment:
Skeptical of using `IntEnum` vs our own `Enum`, don't they functionally operate the same anyways?